### ISSUE TITLE
Can this package be compatible with firebase-admin 6?

### ISSUE THREAD

**Issue by ayyoubelamrani4** [xtrinch/fcm-django#229]
While updating to Django 4.2, I have a warning:

```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
fcm-django 1.0.12 requires firebase-admin<6,>=5, but you have firebase-admin 6.1.0 which is incompatible.
```

I am wondering if we can make this library compatible with firebase-admin 6?

**Comment by xtrinch** [comment]
I do think it should be compatible out of the box with 6, but it needs to be tested and verified, then we can remove the <5 requirement

**Comment by thecodepapaya** [comment]
We are blocked on this issue as we need to use `firebase_admin` 6 with the latest `fcm_django`.
Is there a workaround for this? or a way to contribute and help expedite the process of testing and verification?

**Comment by xtrinch** [comment]
Sure, it is helpful to test and report / open PRs for any incompatibilities

**Comment by thecodepapaya** [comment]
My initial thoughts

1. `fcm-django` uses dependency [firebase-admin>=5,<6](https://github.com/xtrinch/fcm-django/blob/5bbd7d0168ab26221621d6749522b35e0581f56d/requirements.txt#L4)
2. Changelog for firebase-admin v6 contains [two changes](https://github.com/firebase/firebase-admin-python/releases/tag/v6.0.0)
    a. [Major] Dropped support for Python v3.6 (EoL)
    b. [Minor] Introduced app_check.verify_token()
3. Since `fcm-django` is [fully compatible with Python 3.6+](https://github.com/xtrinch/fcm-django#python-3-support), no breaking change should exist.

Can we safely remove the `<6` dependency for firebase-admin? Or are there other things that might need to be tested?

**Comment by xtrinch** [comment]
Released in 2.0.0
